### ISSUE TITLE
Create recurrent meeting with no fixed time

### ISSUE THREAD

**Issue by chiarapivetta** [Jericho/ZoomNet#227]
As specified in the Zoom Api documentation, there are two possible recurrent meeting type in Zoom. One of them being with no fixed time and the other one with fixed time.

It's not clear to me how I would create a recurrence meeting with **no fixed time** using this package. To achive this using the zoom api, you don't set the recurrence object and you specify the type of the meeting (3).
 
I tried calling CreateRecurringMeetingAsync() passing it an empty RecurrenceInfo instance (it can't be null) but it gave me this error:

  _ZoomNet.Utilities.ZoomException: Validation Failed.
         at ZoomNet.Utilities.ZoomErrorHandler.OnResponse(IResponse response, Boolean httpErrorAsException)
         at Pathoschild.Http.Client.Internal.Request.Execute()
         at ZoomNet.Internal.AsObject[T](IRequest request, String propertyName, Boolean throwIfPropertyIsMissing, JsonSerializerOptions options)_

**Comment by Jericho** [comment]
ZoomNet is designed to automatically detect whether your are attempting to create a recurring meeting with or without fixed time based on the presence (or absence) of the recurring information when you invoke `CreateRecurringMeetingAsync`. Therefore, invoking `CreateRecurringMeetingAsync` with a null RecurenceInfo is indeed what you are supposed to do to create a recurring meeting with no fixed time.

However, as you have discovered, there is currently a flaw in the way that we determine whether your meeting has fixed time or not which leads ZoomNet to attempt to create a meeting with fixed time (type=8) but without recurrence info which the Zoom API rejects.

Long story short: I need to fix `CreateRecurringMeetingAsync` so that it properly determines whether your meeting has recurring time or not.

Thanks for reporting this issue.

**Comment by chiarapivetta** [comment]
Thank you! I'll wait for the updated version
